frontend/src/stories/storyMocks.tsx

```diff
@@ -0,0 +1,67 @@
+import { graphql, GraphQLHandler, GraphQLRequest } from "msw";
+import {
+  ApolloClient,
+  HttpLink,
+  InMemoryCache,
+  ApolloProvider,
+} from "@apollo/client";
+
+const mocks = {
```

Comment:
This is the list of all mocks that I'd like to aggregate here for reuse

Reply:
Will we add our REST mocks here as well?

Reply:
I believe adding the REST mocks here would work